test/integration/puppetlabs/puppetserver/bootstrap_testutils.clj

```diff
@@ -49,6 +51,12 @@
         (assoc-in [:jruby-puppet :master-log-dir] master-log-dir)
         (ks/deep-merge overrides))))
 
+(defn services-from-dev-bootstrap-plus-mock-jruby-pool-manager-service
+  [config]
+  (conj (tk-bootstrap/parse-bootstrap-config!
+         jruby-puppet-testutils/dev-bootstrap-file-without-jruby-pool-service)
```

Comment:
Sucks that we have to write a temp file for this, but I think I can see why you did it that way. 

Would it not be possible, though, to just call `parse-bootstrap-config!` directly on the "real" file, and then do a `filter` on the resulting vector to remove the real jruby pool service?